### PR TITLE
STY: PEP8 amendments

### PR DESCRIPTION
**Pull request by jremes-foss**
Few more PEP8 amendments before the weekend. :) 

benchmarks/benchmarks/sparse_linalg_expm.py
benchmarks/benchmarks/sparse_linalg_lobpcg.py

Thank you.

### CODE REVIEW

File: benchmarks/benchmarks/sparse_linalg_lobpcg.py [hunk 1]
```diff
@@ -23,13 +23,13 @@ def _sakurai(n):
...
+    d0 = array(r_[5, 6 * ones(n - 2), 5])
+    d1 = -4 * ones(n)
     d2 = ones(n)
-    B = scipy.sparse.spdiags([d2,d1,d0,d1,d2],[-2,-1,0,1,2],n,n)
+    B = scipy.sparse.spdiags([d2, d1, d0, d1, d2], [-2, -1, 0, 1, 2], n, n)
 
-    k = arange(1,n+1)
-    w_ex = sort(1./(16.*pow(cos(0.5*k*pi/(n+1)),4)))  # exact eigenvalues
+    k = arange(1, n + 1)
+    w_ex = sort(1. / (16. * pow(cos(0.5 * k * pi / (n + 1)), 4)))  # exact eigenvalues
```
Comment (rgommers) [inline benchmarks/benchmarks/sparse_linalg_lobpcg.py]: This doesn't look right, too many spaces. This is the case in multiple places. Please have a look at what PEP8 says about expressions with operators:

    If operators with different priorities are used, consider adding whitespace
    around the operators with the lowest priority(ies)

See also the example at https://www.python.org/dev/peps/pep-0008/#other-recommendations

Can you please correct this whole PR for that?

**Comment by josef-pkt** [comment]
@rgommers   just out of curiousity

Are you switching to "new" pep-8?   In the old times it was around all operators except `**`.

**Comment by rgommers** [comment]
> Are you switching to "new" pep-8? In the old times it was around all operators except ``**``.

I think we've changed already a while ago, gradually. Not a single decision I think, just evolution of how people have been writing code.

**Comment by rgommers** [comment]
Some code analysis tools still do this old-style I think, but the change to PEP8 is already a couple of years old by now.

File: benchmarks/benchmarks/sparse_linalg_lobpcg.py (hunk 1 above)
Comment (Newman101) [inline benchmarks/benchmarks/sparse_linalg_lobpcg.py]: Sure, I can correct this. Thanks for notifying me, I think this was my mistake.

Comment (rgommers) [inline benchmarks/benchmarks/sparse_linalg_lobpcg.py]: @Newman101 are you planning to finish this?

Here's an example for this line of a possible improvement:

     w_ex = sort(1. / (16. * pow(cos(0.5*k*pi / (n+1)), 4)))

There's some personal preference involved in deciding on the exact amount of spaces, but I hope the idea is clear.

**Comment by Newman101** [comment]
I reviewed the standards and looks like these should not have been changed. For example, lines 31 and 32 are fine as they are. White space around operators (at least according to documentation) are the least priority.

**Comment by rgommers** [comment]
@Newman101 correct. The "space after comma" and indentation changes are still useful though.

**Comment by Newman101** [comment]
Sure. If it is ok, I'll amend the incorrect changes and re-commit.

**Comment by rgommers** [comment]
> Sure. If it is ok, I'll amend the incorrect changes and re-commit.

Yes please, that's the preferred way of doing this kind of thing.

**Pull request merged**